packages/tools/examples/stackManipulationTools/index.ts

```diff
@@ -87,7 +88,7 @@ async function run() {
   // Add tools to Cornerstone3D
   cornerstoneTools.addTool(PanTool);
   cornerstoneTools.addTool(WindowLevelTool);
-  cornerstoneTools.addTool(StackScrollMouseWheelTool);
```

Comment:
Note that the stack scroll mouse wheel tool is deprecated - I will make the rest of them go away once you have looked over the basics here.